aptos-move/e2e-move-tests/src/tests/aggregator_v2.data/pack/sources/runtime_checks.move

```diff
@@ -0,0 +1,189 @@
+/// Aggregators and other structs that use delayed fields have certain
```

Comment:
nit: OCD, these comments can fit on two lines. but maybe we should add small sentence about why, i.e. this is because delayed field values get exchanged with unique identifiers by runtime.